neuraxle/base.py

```diff
@@ -280,33 +280,13 @@ def fit_transform(self, data_inputs, expected_outputs=None) -> ('BaseStep', Any)
         return new_self, out
 
     def fit(self, data_inputs, expected_outputs=None) -> 'BaseStep':
```

Comment:
Add "@abstractmethod" to force implementing fit and also raise NotImplementedError.